src/hope_country_report/apps/power_query/processors.py

```diff
@@ -245,52 +252,25 @@ def process(self, context: Dict[str, Any]) -> bytes:
         fitz_pdf_document = fitz.open(stream=output_stream, filetype="pdf")
 
         # Convert the PDF to an image-based PDF
-        image_pdf_bytes = self.convert_pdf_to_image_pdf(fitz_pdf_document, dpi=300)
+        image_pdf_bytes = convert_pdf_to_image_pdf(fitz_pdf_document, dpi=300)
 
         return image_pdf_bytes
 
-    def convert_pdf_to_image_pdf(self, pdf_document: fitz.Document, dpi: int = 300) -> bytes:
-        """
-        Converts each page of a PDF document to an image and then creates a new PDF
-        with these images as its pages.
-        """
-        new_pdf_document = fitz.open()
-
-        for page_num in range(len(pdf_document)):
-            pix = pdf_document[page_num].get_pixmap(dpi=dpi)
-            new_pdf_document.new_page(width=pix.width, height=pix.height)
-            new_pdf_document[page_num].insert_image(fitz.Rect(0, 0, pix.width, pix.height), pixmap=pix)
-        new_pdf_bytes = io.BytesIO()
-        new_pdf_document.save(new_pdf_bytes, deflate_fonts=1, deflate_images=1, deflate=1)
-        new_pdf_bytes.seek(0)
-        return new_pdf_bytes.getvalue()
-
-    def insert_arabic_image(self, document: fitz.Document, field_name: str, text: str):
-        """
-        Generates and inserts an image containing the given Arabic text into
-        the specified field.
-        """
-        rect, page_index = self.get_field_rect(document, field_name)
-        if rect:
-            image_stream = self.generate_arabic_image(text, rect)
-            img_rect = fitz.Rect(*rect)
-            page = document[page_index]
-            page.insert_image(img_rect, stream=image_stream, keep_proportion=False)
-
-    def insert_external_image(self, document: fitz.Document, field_name: str, image_path: str):
+    def insert_external_image(self, document: fitz.Document, field_name: str, image_path: str, font_size: int = 10):
         """
         Loads, resizes, and inserts an external image into the specified field.
         """
-        rect, page_index = self.get_field_rect(document, field_name)
+        rect, page_index = get_field_rect(document, field_name)
         if rect is None or page_index is None:
             logger.error(f"No valid rectangle or page index found for field {field_name}. Cannot insert image.")
             return
         page = document[page_index]
         try:
             image_stream = self.load_image_from_blob_storage(image_path)
-            image = Image.open(image_stream)
+            image = Image.open(image_stream).rotate(-90, expand=True)
+            image.thumbnail((800, 600), Image.LANCZOS)
             output_stream = io.BytesIO()
-            image.save(output_stream, format="JPEG", quality=75)
+            image.save(output_stream, format="PNG")
```

Comment:
why PNG now?

Reply:
less heavy in the final PDF. I had to lower the quality of jpg before.